packages/api-derive/src/elections/approvalsOf.ts

```diff
@@ -2,9 +2,11 @@
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
 
+import { SetIndex } from '@polkadot/types/srml/elections/types';
```

Comment:
This also works as `import { SetIndex } from '@polkadot/srml/types'` however kept it very explicit here.